docs/vsphere_ci/build.json

```diff
@@ -18,7 +18,7 @@
   ],
   "provisioners": [
     {
-      "pause_before": "3m",
+      "pause_before": "10m",
```

Comment:
Do we need to wait that long, even with [this](https://github.com/openshift/windows-machine-config-operator/pull/1052/files#diff-f95d5d0ee98dedf392fb54c0fb93f7e3df3c9699b01d08ca8fd00d30ccba8244R124) execution order?